src/utils/getPlaceholder.js

```diff
@@ -0,0 +1,24 @@
+import {
+  isStreamNarrow,
+  isTopicNarrow,
+  isPrivateNarrow,
+  isGroupNarrow,
+} from '../utils/narrow';
+
+export const getInputPlaceholderFromNarrow = (narrow, auth, users) => {
```

Comment:
1. Wondering if this could be `export default`. But if we decide to use this file for retrieving placeholders for different inputs e.g. Search e.t.c there's no need of that. 
2.  Also considering the names of files already inside `utils/` `placeholder.js`/`placeholders.js` seems like a more reasonable name to me.
3. Why not consider a shorter method name? e.g. `inputPlaceholder()`